ufo2fontir/src/source.rs

```diff
@@ -35,6 +26,59 @@ impl DesignSpaceIrSource {
     }
 }
 
+fn glif_files<'a>(
+    ufo_dir: &Path,
+    layer_cache: &'a mut HashMap<String, HashMap<String, PathBuf>>,
+    source: &designspace::Source,
+) -> Result<BTreeMap<String, PathBuf>, Error> {
+    let layer_name = layer_name(ufo_dir, layer_cache, source)?;
```

Comment:
I would not call this method or local variable `layer_name`, because it's not the _name_ of a layer (that's the _key_ in the layercontents ordered dict), it's the directory path associated with that layer (or the _value_ of layercontents.plist entries).

Reply:
Fixed